README.md

```diff
@@ -14,10 +14,26 @@ Additionally, it provides [Snippets](https://code.visualstudio.com/docs/editor/u
 
 ### VS Code
 
-This extension is not yet published, so you will need to install it manually.
-First, clone and change into this repository: `git clone https://github.com/awslabs/smithy-vscode.git && cd smithy-vscode`
-Then, run npm commands to install:
+This extension is not yet published. To use this extension, manually installing
+the Extension with the following steps:
+* Clone the Extension: `git clone https://github.com/awslabs/smithy-vscode.git && cd smithy-vscode`
+* Run npm commands to install:
 `npm install && npm run install-plugin`
+* Open VS Code and add a `smithy-build.json` file to the root of your project,
```

Comment:
Is this necessary? What happens when I look at a detached model file?

Reply:
Not in that case. Only if you have dependencies that need loaded. I'll update that.